src/sentry/static/sentry/app/views/performance/transactionSummary/trendChart.tsx

```diff
@@ -0,0 +1,235 @@
+          formatter: (value: number) => axisLabelFormatter(value, 'p50()'),
+        },
+      },
+    };
+
+    return (
+      <React.Fragment>
+        <HeaderTitleLegend>
+          {t('Trend')}
+          <QuestionTooltip
+            size="sm"
+            position="top"
+            title={t(`Trends shows the smoothed value of an aggregate over time.`)}
+          />
+        </HeaderTitleLegend>
+        <ChartZoom
+          router={router}
+          period={statsPeriod}
+          projects={project}
+          environments={environment}
+        >
+          {zoomRenderProps => (
+            <EventsRequest
+              api={api}
+              organization={organization}
+              period={statsPeriod}
+              project={[...project]}
+              environment={[...environment]}
+              start={start}
+              end={end}
+              interval={getInterval(datetimeSelection, true)}
+              showLoading={false}
+              query={query}
+              includePrevious={false}
+              yAxis={YAXIS_VALUES}
+            >
+              {({results: _results, errored, loading, reloading}) => {
+                if (errored) {
+                  return (
+                    <ErrorPanel>
+                      <IconWarning color="gray500" size="lg" />
+                    </ErrorPanel>
+                  );
+                }
+
+                const results = _results?.filter(r => r.seriesName === trendDisplay);
+
+                const series = results
+                  ? results
+                      .map(values => {
+                        return {
+                          ...values,
+                          color: theme.purple400,
+                          lineStyle: {
+                            opacity: 0.75,
+                            width: 1,
+                          },
+                        };
+                      })
+                      .reverse()
+                  : [];
+
+                const {smoothedResults} = transformEventStatsSmoothed(
+                  results,
+                  t('Smoothed')
+                );
+
+                const smoothedSeries = smoothedResults
+                  ? smoothedResults.map(values => {
+                      return {
+                        ...values,
+                        color: theme.purple500,
+                        lineStyle: {
+                          opacity: 1,
+                        },
+                      };
+                    })
+                  : [];
+
+                // Stack the toolbox under the legend.
+                // so all series names are clickable.
+                zoomRenderProps.toolBox.z = -1;
+
+                return (
+                  <ReleaseSeries
+                    start={start}
+                    end={end}
+                    period={statsPeriod}
+                    utc={utc}
+                    projects={project}
+                    environments={environment}
```

Comment:
Should this be memoized?

Reply:
Release won't get called again. fetchData doesnt get called again because you aren't changing anything here, you are just switching between data you already have.